mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -85,6 +85,11 @@
     <!-- Text displayed that links to website containing documentation about the "Total cookie protection" feature. -->
     <string name="tcp_cfr_learn_more">Learn about Total Cookie Protection</string>
 
+    <!-- Private browsing erase action "contextual feature recommendation" (CFR) -->
+    <!-- Text for the message displayed in the contextual feature recommendation popup promoting the erase private browsing feature. -->
+    <string name="erase_action_cfr_message">Tap here to start a fresh private session. Delete your history, cookies  — everything.</string>
```

Comment:
```suggestion
    <string name="erase_action_cfr_message">Tap here to start a fresh private session. Delete your history, cookies — everything.</string>
```

@boek 
You have a double space that it's worth fixing here (no need to change the string ID if we don't merge this PR).
From https://github.com/mozilla-mobile/firefox-android/pull/3145

Reply:
I'm going to temporarily remove the space in this PR, but this needs to be fixed in code, as it will reappear on the next import.